nbs/_quarto.yml

```diff
@@ -33,10 +33,10 @@ website:
     left:
       - text: "Get Started"
         href: getting_started.ipynb
-      - text: "Tutorial"
+      - text: "Tutorials"
```

Comment:
This is a link to the actual tutorial, so `Tutorial` was correct. Unless we think linking to the tutorials index is better, then below line should become `href: tutorials`

Reply:
oh I see good catch, I misunderstood

Reply:
ok reverted this 👍🏽